hist/hist/src/THStack.cxx

```diff
@@ -1071,12 +1063,12 @@ void THStack::SavePrimitive(std::ostream &out, Option_t *option /*= ""*/)
          TString hname = h->GetName();
```

Comment:
Doesn't this need to be transformed into a valid variable name?

Reply:
I not yet touch this code.

Maybe one need other method to create unique and valid identifier for such cases

Reply:
We do have `ROOT::TMetaUtils::GetCppName` that could/should be used for all `Name to variablename` transformation.  (This change could indeed be in separate PR).

Reply:
it should be not only valid name, but also unique name. 
I prefer to change it like for other classes.